resources/js/data-access/memo-api/tquery/table.ts

```diff
@@ -161,7 +169,7 @@ export function createTableRequestCreator({
       requestController: {
         columnVisibility: [columnVisibility, setColumnVisibility],
         globalFilter: [globalFilter, setGlobalFilter],
-        columnFilter: (column) => columnFilters()[column]!,
+        columnFilter,
```

Comment:
Nit: Generalnie ten poprzedni zapis z arrow wydawał mi się czytelniejszy - nie musiałem w tym miejscu pamiętać że to `columnFilter` ro funkcja która dopiero zwraca filter dla danej kolumny.

Może zmienić nazwę funkcji na coś w stylu `getColumnFilter(column)` i tu zostawić formę arrow funkcji? Albo zmienić też nazwę w requestController (choć to więcej roboty) - ale to pole ma inne działanie niż pozostałe - to nie sygnał tylko funkcja zwracająca sygnał.